core/src/main/java/com/linecorp/armeria/server/AbstractPathMapping.java

```diff
@@ -42,6 +42,10 @@ static String mappedPath(String prefix, String path) {
         return path.substring(length);
     }
 
+    static boolean hasQueryString(String path) {
```

Comment:
Question) For reference, is there a reason `PrefixPathMapping` doesn't apply this check?

Reply:
It should be find to add the validation. Since a prefix is expected to contain additional path segments, having a query string there would not be logical. So I though it would not cause confusion unlike a regular path.

However, if adding it is meaningful, I will include it.

Reply:
+1 for adding it.

Reply:
Fixed at [`7e06404` (#6546)](https://github.com/line/armeria/pull/6546/commits/7e06404f110f5ee394cc5ab3a582812e76929206)